desc/compute/_geometry.py

```diff
@@ -155,45 +158,20 @@ def _V_rrr_of_r(params, transforms, profiles, data, **kwargs):
     label="A(\\zeta)",
     units="m^{2}",
     units_long="square meters",
-    description="Cross-sectional area as function of zeta",
+    description="Area of enclosed cross-section (enclosed constant phi surface), "
+    "scaled by max(ρ)⁻², as function of zeta",
     dim=1,
     params=[],
     transforms={"grid": []},
     profiles=[],
     coordinates="z",
-    data=["|e_rho x e_theta|"],
+    data=["Z", "n_rho", "e_theta|r,p", "rho"],
     parameterization=[
         "desc.equilibrium.equilibrium.Equilibrium",
         "desc.geometry.surface.ZernikeRZToroidalSection",
+        "desc.geometry.surface.FourierRZToroidalSurface",
     ],
-    resolution_requirement="rt",
-)
-def _A_of_z(params, transforms, profiles, data, **kwargs):
-    data["A(z)"] = surface_integrals(
```

Comment:
When the grid is truncated to θ ∈ [0, π], this surface integral give more accurate results than the line integral method, sometimes significantly. It will also give the correct result for a broader range of surfaces, for example `test_elongation` on master passes using the surface integral but not with a line integral. However, I think it's still wrong in general when truncated θ ∈ [0, π] just like the line integral method, so I removed it in favor of the line integral for the purpose of this PR's title.

Reply:
So, as documented with the numerical differences in commit [be11015](https://github.com/PlasmaControl/DESC/pull/1094/commits/be1101524a11f49d293b1661c44b2b67a5efd747), the new master compute data for 'A(z)' in this PR is less correct than on `master`.